pyplanet/apps/contrib/local_records/__init__.py

```diff
@@ -299,7 +299,9 @@ async def chat_current_record(self):
 		if records_amount > 0:
 			first_record = self.current_records[0]
 			message = '$0f3Current Local Record: $fff\uf017 {}$z$s$0f3 by $fff{}$z$s$0f3 ($fff{}$0f3 records)'.format(
-				times.format_time(first_record.score), first_record.player.nickname, records_amount
+				times.format_time(first_record.score),
+				(lambda x: x if not (x[-1] == '$' and x[-2:].count('$') == 1) else x[:-1])(first_record.player.nickname),
```

Comment:
(Same question as before.)

Reply:
(Same answer as before.)

@tomvlk 

For local_records __init__.py might not be the best location - I am not sure why I did put it there, but since the player name will be derived from the database, it might be better to save the nickname to the database without a leading $ if there is one. Then there has to be a update player check with adjusted nickname too.